config/device-classes/generic/vmware-esxi.yaml

```
@@ -0,0 +1,80 @@
+name: "vmware-esxi"
+
+config:
+  components:
+    cpu: true
+    memory: true
+    disk: true
+    server: true
+
+match:
+  logical_operator: OR
+  conditions:
+    - type: SysObjectID
+      match_mode: startsWith
+      values:
+        - ".1.3.6.1.4.1.6876.4.1"
+    - type: SysDescription
+      match_mode: regex
+      values:
+        - '^VMware ESXi'
+
+identify:
+  properties:
+    vendor:
+      - detection: constant
+        value: "VMware"
+    model:
+      - detection: SysDescription
```

Same as in the Clavister device class, please remove the vendor from the model.
```suggestion
      - detection: SysDescription
        operators:
          - type: modify
            modify_method: regexSubmatch
            regex: '^VMware (.+)'
            format: "$1"
```